src/Textarea/Textarea.scss

```diff
@@ -1,4 +1,4 @@
-@import "../../sass/base/index";
```

Comment:
Superb! Super minor change, could avoid rewriting quotes since the rest of the codebase uses double quotes?

Reply:
Could you either change this, or contribute a Prettier configuration file that describes the style of the present code @adneforslund ? https://prettier.io/docs/en/configuration.html